src/components/plot/add/PlotAddChoosePlotter.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useFormContext } from 'react-hook-form';
+import { t, Trans } from '@lingui/macro';
+import { CardStep, Select } from '@chia/core';
+import {
+  FormControl,
+  FormHelperText,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Typography,
+} from '@material-ui/core';
+import { RootState } from '../../../modules/rootReducer';
+import PlotterName from '../../../constants/PlotterNames';
+import StateColor from '../../core/constants/StateColor';
+import styled from 'styled-components';
+import { defaultPlotter } from '../../../modules/plotterConfiguration';
+
+type Props = {
+  step: number;
+  onChange: (plotter: PlotterName) => void;
+};
+
+const StyledFormHelperText = styled(FormHelperText)`
+  color: ${StateColor.WARNING};
+`;
+
+export default function PlotAddChoosePlotter(props: Props) {
+  const { step, onChange } = props;
+  const { watch } = useFormContext();
+  const plotterName: PlotterName = watch('plotterName');
+  const { availablePlotters } = useSelector((state: RootState) => state.plotter_configuration);
+  const displayedPlotters = Object.keys(availablePlotters) as PlotterName[]
+  
+  // Sort chiapos to the top of the list
+  displayedPlotters.sort((a, b) => a == PlotterName.CHIAPOS ? -1 : a.localeCompare(b));
+
+  const handleChange = async (event: any) => {
+    const selectedPlotterName: PlotterName = event.target.value as PlotterName;
+    onChange(selectedPlotterName);
+  };
+
+  const isPlotterInstalled = (plotterName: PlotterName): boolean => {
+    const installed = availablePlotters[plotterName]?.installInfo?.installed ?? false;
+    return installed;
+  }
+
+  const isPlotterSupported = (plotterName: PlotterName): boolean => {
+    const installed = availablePlotters[plotterName]?.installInfo?.installed ?? false;
+    const supported = installed || (availablePlotters[plotterName]?.installInfo?.canInstall ?? false);
+    return supported;
+  }
+
+  const plotterDisplayName = (plotterName: PlotterName): string => {
```

Comment:
just comment: use a standard function instead of arrow function. we can see more information in the stacktrace. use arrow function only for callbacks (arguments)
instead
const plotterDisplayName = () =>

function plotterDisplayName() {}